.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+on:
+  push:
+  pull_request:
+    branches:
+      # Branches from forks have the form 'user:branch-name' so we only run
+      # this job on pull_request events for branches that look like fork
+      # branches. Without this we would end up running this job twice for non
+      # forked PRs, once for the push and then once for opening the PR.
+    - '**:**'
+
+jobs:
+  ci:
+    strategy:
+      matrix:
+        include:
+          - otp-version: 23.1
+            platform: ubuntu-20.04
+            lsb_release: focal
+          - otp-version: 22.3.4.9
+            platform: ubuntu-20.04
+            lsb_release: focal
+          - otp-version: 21.3.8.17
+            platform: ubuntu-20.04
+            lsb_release: focal
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Cache Hex packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/rebar3/hex/hexpm/packages
+          key: ${{ runner.os }}-hex-${{ hashFiles('**/rebar.lock') }}
+          restore-keys: ${{ runner.os }}-hex-
```

Comment:
I don't think we're using any Hex packages in this repo.

Reply:
Sorry, we do for Poolboy. (I mixed up the review with another review I had open.)